### PR TITLE
Enable drone build

### PR DESCRIPTION
**Pull request by easternbloc**
- This will run unit tests
- Build the dockerfile
- Publish to quay

### CODE REVIEW

File: .drone.yml [hunk 1]
```diff
@@ -0,0 +1,30 @@
+pipeline:
+  build_5:
```
Comment (JoeChapman) [inline .drone.yml]: why call this `build_5`?

Comment (easternbloc) [inline .drone.yml]: good 👁

**Comment by easternbloc** [comment]
@JoeChapman updated

**Comment by easternbloc** [comment]
now with added docker service!

File: .drone.yml [hunk 2]
```diff
@@ -0,0 +1,37 @@
+pipeline:
+  build_6:
+    image: node:6
+    commands:
+      - npm --loglevel warn install
+      - npm run lint
+      - npm run test
+    when:
+      event: [push, pull_request]
```
Comment (JoeChapman) [inline .drone.yml]: is this running tests on `push` against what? The app builds on `pull_request`s. Or am I missing something?

Comment (easternbloc) [inline .drone.yml]: this will run against . basically so the tests in file-vault. we use this pattern everywhere

**Pull request merged**